book/integration-service.md

```diff
@@ -0,0 +1,207 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of controllers that facilitate automated testing of content produced by the build pipelines. It is mainly responsible for integration testing capabilities.
+
+The Integration service uses the pipeline, snapshot, and environment controllers to watch for Component builds, trigger and manage testing Tekton Pipelines, and create releases based on the testing outcome(s).
+
+### Goals
+
+- The Integration service need to be able to deploy a specific set of images to an environment and record the results of integration testing.
+- Given a specific set of images, Integration service should be able to tell if they passed application validation tests.
+- The Integration service should be able to update one component image at a time in order to support 'slow loop' application validation testing.
```

Comment:
You don't explain what slow loop is. I know we removed descriptions of this earlier, but I think we need some explanation if you are going to use the terminology.

Reply:
We can change this to `The Integration service should be able to update one component image at a time in order to test and deploy individual component builds.`
We will later update this with a bit more details or references as part of [HACBS-1426](https://issues.redhat.com/browse/HACBS-1426).

Reply:
Done